src/rgw/rgw_sal_rados.cc

```diff
@@ -3019,6 +3019,11 @@ int RadosZoneGroup::get_zone_by_name(const std::string& name, std::unique_ptr<Zo
   return 0;
 }
 
+int RadosZoneGroup::list_zones(const DoutPrefixProvider* dpp, std::list<std::string>& zone_ids)
+{
+  return store->svc()->zone->list_zones(dpp, zone_ids);
```

Comment:
this lists all zones in the cluster, right? i'd expect a `ZoneGroup::list_zones()` to only list the zones in that zonegroup. maybe we need a better name or place for this?